content/deploy/choose_cloud.md

```diff
@@ -0,0 +1,70 @@
+# Choosing where your JupyterHub will run
+
+## Should I deploy on my hardware or in the cloud?
+
+The first thing you'll decide is *where* your JupyterHub will run. The first
+question to ask is "will I run my JupyterHub on my own hardware, or will I
+run it in the cloud?" Running a JupyterHub on your own hardware is your best
+option if you have some technical support for doing so. If you're working with
+limited technical resources, running a JupyterHub in the cloud is usually simpler.
+The rest of this guide assumes you'd like to run a JupyterHub in the cloud.
+
+(however, note that you can still [demo the course environment locally](deploy_locally.html))
+
+## Choosing a cloud provider and service
+
+Cloud providers (e.g. Microsoft Azure, Google Cloud, or Digital Ocean) are where you'll
+run your JupyterHub. These companies basically own really large computers, and what
+you do is purchase some resources on those computers to run whatever you'd like (in this
+case, a JupyterHub). However, there are often a few different kinds of offerings that
+cloud providers provide, with various pros and cons.
+
+> **An aside on using free hosted services vs. creating your own deployments**. Many cloud
+  providers offer a "free" and customized notebook service for basic usage. While these are
+  appealing on their surface, they provide a number of challenges down the road, such as
+  getting your students used to the vendor-specific interface rather than an open-source version.
+  The Jupyter Project (and the Data 8 approach) aims to maximize your choice and flexibility in
+  how and where you deploy Jupyter environments on shared infrastructure, and for this reason
+  we do not recommend using most fully-managed "free" services.
+
+With that in mind, here are a few of the most-common options for choosing your cloud resources.
+
+* **You purchase computational resources and set up yourself**. The recommmened approach. In this case, you ask the
+  cloud provider for some resources, and they give you a blank slate that you can do anything
+  with. This is the most-common approach to deploying JupyterHub (along with
+  following this guide to do the actual set-up).
+
+  * **Pros**: You have flexibility in customizing your JupyterHub exactly as you wish. You also
+    have the ability to easily switch to other cloud providers because the setup process will
+    be very similar for each.
+  * **Cons**: There is more manual setup and maintenance for you to perform. Because the resources
+    are a blank slate, you'll need to take the steps to set everything up.
+* **You use a "push-button" deployment to deploy specific resources**. Another possibility is that
```

Comment:
Does this actually exist anywhere yet? I haven't seen a "push button" approach by any of the cloud providers yet. Are there examples we could point to? Otherwise, this discussion is quite abstract and not necessarily helpful for an instructor looking to get started.